### DISCUSSION TITLE
Is there a way to limit tabs by permission

### DISCUSSION THREAD

**Discussion by rappasoft** [filamentphp/filament#8631] [General]
I have a resource with X tabs on the list page.

Is there a way to lock these tabs down by permission, whatever permission you have lets you see and interact with those tabs, as well as the left navigation is active if they can see at least 1 tab?

I went through the API and don't see functionality in the Tab class, wondering if anyone has tacked this before.

I'd rather not have to make 4 different resources.

Thanks

**Comment by wychoong** [comment]
getTabs just returning an array. Can’t you just filter or only add the tab to the array if the user have permission to see it?

**Reply by rappasoft** [reply]
Thats.....a good idea and I'm mad I overthought it.

Thanks!
